docs/API/LorisRESTAPI_v0.0.3.md

```diff
@@ -759,42 +759,46 @@ object of the form:
         "Visit" : $VisitLabel,
     },
     "DicomTars" : 
-        [{
-        "Tarname" : "DCM_yyyy-mm-dd_ImagingUpload-hh-mm-abc123.tar",
-        "SeriesInfo" :
-            [{
-            "SeriesDescription" : "MPRAGE_ipat2",
-            "SeriesNumber" : "2",
-            "EchoTime" : "2.98",
-            "RepetitionTime" : "2300",
-            "InversionTime" : "900",
-            "SliceThickness" : "1",
-            "Modality" : "MR",
-            "SeriesUID" : "1.2.3.4.1107",
-            },
-            {
-            "SeriesDescription" : "BOLD Resting State",
-            "SeriesNumber" : "5",
-            "EchoTime" : "30",
-            "RepetitionTime" : "2100",
-            "InversionTime" : NULL,
-            "SliceThickness" : "3.5",
-            "Modality" : "MR",
-            "SeriesUID" : "3.4.5.6.1507",
-            }],
-        "Tarname" : "DCM_yyyy-mm-dd_ImagingUpload-hh-mm-def456.tar",
-        "SeriesInfo" :
-            [{
-            "SeriesDescription" : "MPRAGE_ipat2",
-            "SeriesNumber" : "2",
-            "EchoTime" : "2.98",
-            "RepetitionTime" : "2300",
-            "InversionTime" : "900",
-            "SliceThickness" : "1",
-            "Modality" : "MR",
-            "SeriesUID" : "1.7.8.9.1296",
-            }],
-    }],    
+    [
+        {
+            "Tarname" : "DCM_yyyy-mm-dd_ImagingUpload-hh-mm-abc123.tar",
+            "SeriesInfo" :
+                [{
+                    "SeriesDescription" : "MPRAGE_ipat2",
+                    "SeriesNumber" : "2",
+                    "EchoTime" : "2.98",
+                    "RepetitionTime" : "2300",
+                    "InversionTime" : "900",
+                    "SliceThickness" : "1",
+                    "Modality" : "MR",
+                    "SeriesUID" : "1.2.3.4.1107",
+                    },
+                    {
+                    "SeriesDescription" : "BOLD Resting State",
+                    "SeriesNumber" : "5",
+                    "EchoTime" : "30",
+                    "RepetitionTime" : "2100",
+                    "InversionTime" : NULL,
+                    "SliceThickness" : "3.5",
+                    "Modality" : "MR",
+                    "SeriesUID" : "3.4.5.6.1507",
+                }],
```

Comment:
That comma should not be there

Reply:
Are trailing commas not allowed?

Reply:
@HenriRabalais JSON specification requires no trailing commas, so returning a JSON object which contains a trailing comma could result in a parse error or invalid JSON response, etc.